src/interpreter/InterpretIfThenElse.ts

```diff
@@ -0,0 +1,75 @@
+import { CommonModel } from '../models/CommonModel';
+import {
```

Comment:
Are you sure you want this file, and not just keeping the current code in `interpreter`? 🤔 Or is it to allow better testing?